static-lang-word-lists/README.md

```diff
@@ -0,0 +1,26 @@
+# static-lang-word-lists
+
+## Motivation
+
+Include word lists in the binary, don't take up more space than necessary, be publishable on crates.io (10 MiB size limit)
+
+## Solution
+
+A build script that downloads the word lists from GitHub, compresses them with Brotli, and embeds that data in the binary, lazily decompressed at runtime
+
+### Technical details
+
+Note: adding or removing a wordlist requires that `egg.py` be re-run
+
+1. A list of all the supported wordlists is generated by `egg.py` (this step is manual and be run when adding or removing a word list!). Their names are generated based on their path - `data/diffenator/latin.txt` becomes `DIFFENATOR_LATIN` in the end crate
+2. The build script uses the code generated by `egg.py` (called `chicken.rs`) to construct URLs to download files
```

Comment:
Not sure the whimsy makes it clearer or muddier, egg already has a meaning in Python: https://stackoverflow.com/questions/2051192/what-is-a-python-egg